server/graphql/v2/object/Host.js

```diff
@@ -313,6 +315,38 @@ export const GraphQLHost = new GraphQLObjectType({
           }
         },
       },
+      stripeIssuingBalance: {
```

Comment:
I'm wondering if we should introduce a dedicated object to retrieve various Stripe related stuff.

```
host {
  id
  stripe {
   issuingBalance {
     amount
     currency
  }
}
```

Reply:
I agree with this and this is consistent with the `Account.transferwise` property that resolves an incapsulated Transferwise object.
Although I understand there's also a `transferwiseBalance` resolver in host, this should ultimately be moved to the Transferwise object.